cluster/mesos/docker/km/build.sh

```diff
@@ -67,7 +67,7 @@ echo "Copying files to workspace"
 mkdir -p "${workspace}/bin"
 #cp "${script_dir}/bin/"* "${workspace}/bin/"
 cp "${common_bin_path}/"* "${workspace}/bin/"
-cp "${kube_bin_path}/"* "${workspace}/bin/"
+cp "${kube_bin_path}/km" "${workspace}/bin/"
```

Comment:
This is only our docker-compose cluster for now. But of course you are right if we want to test such a kubectl serving functionality in this setup.